lib/common_test/src/ct_ssh.erl

```diff
@@ -146,6 +171,16 @@ data for `KeyOrName`. The extra options override any existing options with the
 same key in the configuration data. For details on valid SSH options, see
 application [`SSH`](`e:ssh:index.html`).
 """.
+-spec connect(KeyOrName, ConnType, ExtraOpts) -> {'ok', Handle} | {'error', Reason}
+              when KeyOrName :: atom(),
+                   ConnType :: connection_type(),
+                   ExtraOpts :: [ExtraOption],
+                   ExtraOption :: {'ssh', SSHAddr} | {'sftp', SFTPAddr} | ssh:client_option()
+                                | ssh_sftp:sftp_option(),
+                   SSHAddr :: string() | inet:ip_address() | 'loopback',
```

Comment:
this seems to be definiton of `ssh:host` why it is not used for SSHAddr also?
Is it worth having SSHAddr and SFTPAddr if they're the same type?

Reply:
sftp is just a service run over ssh so I think  {'ssh', ssh::host()} | {'sftp', ssh::host()}

Reply:
yes with a single `:` of course

Reply:
You're right, I don't know why I missed this sorry, will fix.

Reply:
Fixed.